include/mqt-core/dd/Package.hpp

```diff
@@ -2006,24 +1961,18 @@ template <class Config> class Package {
   ///
   mEdge reduceAncillae(mEdge& e, const std::vector<bool>& ancillary,
                        const bool regular = true) {
+    // TODO:these methods do not handle identities yet
```

Comment:
This is also a valid TODO. I believe even the top-level logic currently in the PR does not really properly handle the situation and the version from the main branch is much closer to a full solution (at least here; not for the recursive function).